zeth_contracts/contracts/Pairing.sol

```diff
@@ -149,13 +152,21 @@ library Pairing {
             // any number of pairs (g1, g2) \in G1 x G2.  To check something in
             // the form: e(g1, g2) = e(g'1, g'2), we need to call the
             // precompiled bn256Pairing on input [(g1, g2), (neg(g'1), g'2)]
-            success := call(sub(gas, 2000), 8, 0, add(input, 0x20), mul(inputSize, 0x20), out, 0x20)
+            success := call(
+              sub(gas, 2000),
```

Comment:
Seems like you used 2 spaces for the indent, should be 4

Reply:
Added extra indent.

Reply:
Thanks